src/overlaybd/tar/erofs/liberofs.h

```diff
@@ -0,0 +1,16 @@
+#ifndef TAREROFS_INTERFACE_H
+#define TAREROFS_INTERFACE_H
+
+#include <photon/fs/filesystem.h>
+#include <photon/fs/fiemap.h>
+#include <photon/common/string_view.h>
+
+class LibErofs{
+public:
+    LibErofs(photon::fs::IFile *target);
+    ~LibErofs();
+    int extract_tar(photon::fs::IFile *source, uint64_t blksize, bool meta_only, bool first_layer);
```

Comment:
`uint64_t blksize` should be moved to the constructor.

Reply:
Done.